src/app/beer_garden/db/mongo/api.py

```diff
@@ -55,7 +56,11 @@
 _model_map = {}
 for model_name in beer_garden.db.mongo.models.__all__:
     mongo_class = getattr(beer_garden.db.mongo.models, model_name)
-    _model_map[mongo_class.brewtils_model] = mongo_class
+
+    brewtils_model = getattr(mongo_class, "brewtils_model", None)
+
+    if brewtils_model:
```

Comment:
I had to add this check because I didn't want the new models to inherit from `MongoModel`, at least until I find a compelling reason for them to do so.  Since they don't, and they don't have a corresponding brewtils_model, this check prevents the `_model_map` assignment from just blowing up.  Having this check shouldn't hurt anything either way.

Reply:
Mostly this comes down to me still not seeing the value in using the brewtils models in beer garden.  If using the brewtils models doesn't add any value that I can identify, I'd rather just not use them for now and change to using them if I have to down the line, rather than start using them now for fear that there *is* value, but it's just not apparent to me.

So once I decided not to use the brewtils model, I can't use `MongoModel` as is.  And when I look at `MongoModel` the only thing of possible value it provides is the `index_names` helper.  I might end up needing that later and having to revisit it, but again I'd rather make the adjustments when / if the use becomes apparent.